aiosmtpd/tests/test_handlers.py

```diff
@@ -407,14 +407,17 @@ def test_deliver(self):
                 'anne@example.com', ['bart@example.com'], self.message)
             client.quit()
             mock().connect.assert_called_once_with('localhost', 9025)
+            # SMTP always fixes eols, so it must be always CRLF as delimeter
+            msg = NEWLINE.join([
+                'From: Anne Person <anne@example.com>',
+                'To: Bart Person <bart@example.com>',
+                'Subject: A test',
+                'X-Peer: ::1',
+                '',
+                'Testing',
+            ])
```

Comment:
Okay, I figured out why that isn't getting warned for `tox -e qa`.  It's up to you whether you want to change this in your branch or not; I'm going to submit a new PR which tightens up this coding style and fixes a few other violations.  (I won't merge it until I rebase it onto this branch, once it lands.)